rosdep/base.yaml

```diff
@@ -4941,6 +4941,7 @@ libturbojpeg:
   gentoo: [media-libs/libjpeg-turbo]
   nixos: [libjpeg_turbo]
   openembedded: [libjpeg-turbo@openembedded-core]
+  opensuse: [libturbojpeg0]
```

Comment:
Well, I've used this libturbojpeg key to build pretty much a full desktop and nothing complained so I don't know if they need the development headers. You can see the packages being built on the OpenSUSE Build Service. 
https://build.opensuse.org/project/show/home:neotinker3:ROS:melodic_test

If development headers are required, then this key should *probably* be defined as "libjpeg8-devel" since it contains the header for libturbojpeg and it will pull in libturbojpeg0 if it isn't installed. 

```
neotinker@ros-15_2:~> rpm -q --requires libjpeg8-devel
/usr/bin/pkg-config
libjpeg8 = 8.1.2
libturbojpeg0 = 8.1.2
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(PayloadIsXz) <= 5.2-1
neotinker@ros-15_2:~> 
```

I say probably because both libjpeg62-devel and libjpeg8-devel have the header for turbojpeg but you can only install one of those packages because they conflict.

Reply:
:thinking: With no other context, version 8 looks like the one used in the ubuntu keys, so if the decision is between 62 and 8, probably 8 is safe. I don't know enough to say whether you need the -devel ones though.